### PR TITLE
fix: update to use axe-core 4.5.2

### PR DESCRIPTION
**Pull request by michael-siek**

### CODE REVIEW

File: packages/react/examples/shadow-dom/package-lock.json [hunk 1]
```diff
@@ -9,7 +9,6 @@
       "version": "4.5.1",
       "license": "MIT",
       "dependencies": {
-        "@axe-core/react": "^4.5.1",
```
Comment (Zidious) [inline packages/react/examples/shadow-dom/package-lock.json:12]: axe-core appears to be removed rather than updated to 4.5.2

**Review by Zidious — CHANGES_REQUESTED** [review]
Minor comment about the React shadow dom example package otherwise LGTM

File: packages/react/examples/shadow-dom/package-lock.json (hunk 1 above)
Comment (michael-siek) [reply packages/react/examples/shadow-dom/package-lock.json:12]: axe-core isn't a dependency of this package

Comment (Zidious) [reply packages/react/examples/shadow-dom/package-lock.json:12]: Oh my mistake

**Review by Zidious — APPROVED** [review]

**Pull request merged**